tools/sys_analyzer/sys_analyzer_linux.py

```diff
@@ -0,0 +1,704 @@
+#!/usr/bin/python3
+"""
+
+Copyright (c) 2019 Intel Corporation All Rights Reserved.
+
+THESE MATERIALS ARE PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL INTEL OR ITS
+CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THESE
+MATERIALS, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+File Name: sys_analyzer_linux.py
+
+Abstract: Script for checking the environment for Media SDK,
+SDK for OpenCL Applications install and run readiness
+
+Notes: Run ./sys_analyzer_linux.py or ./sys_analyzer_linux.py -v
+
+"""
+
+import os, sys, platform
+import os.path
+import pkgconfig # To install: "python3 -m pip install pkgconfig" or "pip3 install pkgconfig"
```

Comment:
Please, add `tools/sys_analyzer/README.md` and describe:
1. Prerequisites for the script
2. Which checks are performed by the script

Also, add a note about the script to top-level README.md